backend/src/v2/compiler/argocompiler/dag.go

```diff
@@ -203,6 +204,10 @@ func (c *workflowCompiler) task(name string, task *pipelinespec.PipelineTaskSpec
 			if inputs.iterationIndex == "" {
 				driver.Depends = depends(task.GetDependentTasks())
 			}
+			// Handle exit handler dependency
+			if task.TriggerPolicy.GetStrategy().String() == "ALL_UPSTREAM_TASKS_COMPLETED" {
```

Comment:
nit: be consistent on using get method or using field directly.

Reply:
done